starport/templates/typed/typed.go

```diff
@@ -44,5 +43,9 @@ func Box(box packd.Walker, opts *Options, g *genny.Generator) error {
 	g.Transformer(genny.Replace("{{typeName}}", opts.TypeName.Snake))
 
 	// Create the 'testutil' package with the test helpers
-	return testutil.Register(g, opts.AppPath)
```

Comment:
Why? This is a more concise (and readable form). Please revert.

Reply:
I am submitting a new PR, which contains only changes to the template.